packages/nuxi/src/commands/dev.ts

```diff
@@ -63,8 +63,8 @@ export default defineNuxtCommand({
       hostname: args.host || args.h || process.env.NUXT_HOST || config.devServer.host,
       https: (args.https !== false && (args.https || config.devServer.https))
         ? {
-            cert: args['ssl-cert'] || (typeof config.devServer.https !== 'boolean' && config.devServer.https.cert) || undefined,
-            key: args['ssl-key'] || (typeof config.devServer.https !== 'boolean' && config.devServer.https.key) || undefined
+            cert: args['ssl-cert'] || process.env.NITRO_SSL_CERT || (typeof config.devServer.https !== 'boolean' && config.devServer.https.cert) || undefined,
+            key: args['ssl-key'] || process.env.NITRO_SSL_KEY || (typeof config.devServer.https !== 'boolean' && config.devServer.https.key) || undefined
```

Comment:
We might in addition, support `NUXT_` variables similar to `NUXT_PORT` and `NUXT_PORT` and also NITRO_ ones for upstream docs compat.

Reply:
Would make sense to support `NUXT_` as long as we also support those at runtime.

Reply:
I guess for Node, yes (if not, we might implement alt env name support). I think regardless we can support it here and _fix_ in upstream.